python/conda/meta.yaml

```diff
@@ -20,12 +19,15 @@ requirements:
   build:
     - python
     - cython >=0.13
+    - nomkl # [not win]
```

Comment:
Is there a reason to require nomkl? I have been running mkl builds without any problems for months now I think...